test/duct/core_test.clj

```diff
@@ -74,6 +74,12 @@
 (defmethod ig/init-key ::bar [_ {:keys [x]}]
   #(update % ::x (fnil conj []) x))
 
+(defmethod ig/init-key :duct.module/routes [_ {:route/keys [resource]}]
```

Comment:
Could you name this key `::module` instead? e.g.

```clojure
(defmethod ig/init-key ::module [_ {:keys [x]}]
  #(assoc-in % [::foo] x))
```

When I read the test initially, it wasn't obvious the `:duct.module/routes` key was one written for the test.